ng-dev/pr/merge/merge-pull-request.ts

```diff
@@ -134,3 +133,14 @@ async function createPullRequestMergeTool(flags: PullRequestMergeFlags) {
     throw e;
   }
 }
+
+/**
+ * Parses the pull request number from either the number or url string
+ */
+export function parsePrNumber(prUrlOrNumber: string): number {
+  const prNumber = parseInt(prUrlOrNumber.split('/').pop()!);
```

Comment:
We should at least have a comment noting that this just takes whatever the result is from `parseInt` on the last segment.

Technically, could do `thisis123` and it would result in `123`, this isnt validating that its a github link for the repo or anything.  Not a big deal, I just want to make sure that its noted what its doing.

Reply:
Well, it's still splitting on `/`. So it wouldn't result in 123 in that case. It would still be not a number and fail. But I can add a comment explaining this.

Reply:
Comment added. We could also move this function to a utils file and use it any time PR numbers are provided as a param, but that'll be a future refactor.